Formula/gaudi.rb

```diff
@@ -0,0 +1,61 @@
+class Gaudi < Formula
+  desc "Provides necessary interfaces and services for building HEP exp frameworks"
+  homepage "https://gaudi.web.cern.ch/gaudi/"
+  url "https://gitlab.cern.ch/gaudi/Gaudi/-/archive/v35r0/Gaudi-v35r0.tar.gz"
+  sha256 "c01b822f9592a7bf875b9997cbeb3c94dea97cb13d523c12649dbbf5d69b5fa6"
+  license "Apache-2.0"
+  head "https://gitlab.cern.ch/gaudi/Gaudi.git"
+
+  depends_on "cmake" => :build
+  depends_on "aida-header"
+  depends_on "boost"
+  depends_on "boost-python3"
+  depends_on "clhep"
+  depends_on "cppunit"
+  depends_on "doxygen"
+  depends_on "gperftools"
+  depends_on "heppdt2"
+  depends_on "jemalloc"
+  depends_on "python@3.9"
+  depends_on "root"
+  depends_on "tbb"
+  depends_on "xerces-c"
+
+  def install
+    # Fix cmake issue with Foundation framework
+
+    inreplace "GaudiKernel/CMakeLists.txt", /target_link_libraries\(GaudiKernel PUBLIC Foundation\)/,
+                        "target_link_libraries(GaudiKernel PUBLIC ${Foundation_FRAMEWORK})"
```

Comment:
This should be reported upstream

Reply:
I has been reported by email. I don't have any git issue to link at the moment